CODEOWNERS

```diff
@@ -19,7 +19,7 @@ extensions/filters/common/original_src @snowp @klarose
 # jwt_authn http filter extension
 /*/extensions/filters/http/jwt_authn @qiwzhang @lizan
 # grpc_http1_reverse_bridge http filter extension
-/*/extensions/filters/http/grpc_http1_reverse_bridge @snowp
```

Comment:
Nice catch.

Can you remove grpc_http1_reverse_bridge from UNOWNED_EXTENSIONS in check_format.py as well?

Reply:
✅